docs/changelog.md

```diff
@@ -226,8 +235,8 @@ tsrc init --groups foo bar
 
 # v0.6.3 (2018-11-04)
 
-* GitHub organization is now `TankerHQ`
```

Comment:
Whoops, looks like this one shouldn't have been replaced...

Reply:
Indeed. Good catch.

Reply:
Fixed in 65ad491

Reply:
Oups, sorry. I thought that I caught all places where it should not be replaced... Thanks!